src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -139,6 +139,11 @@ public new CompressionLevelWithAuto RequestBodyCompressionLevel
         /// </summary>
         public TimeSpan AnrTimeout { get; set; } = TimeSpan.FromSeconds(5);
 
+        /// <summary>
+        /// Whether the SDK should automatically filter `Bad Gateway Exceptions` caused by Unity.
```

Comment:
Above it indicates that it does this only work for iOS. Should we indicate this in the docs here?
```suggestion
        /// Whether the SDK should automatically filter `BadGatewayException` caused by Unity on iOS.
```

Reply:
There are a whole bunch of places these exceptions are coming from. We already filter them on the C# layer but now, with the same flag because it's the same noise, we also filter the ones coming from native iOS.